EKF/gps_yaw_fusion.cpp

```diff
@@ -277,8 +277,8 @@ bool Ekf::resetGpsAntYaw()
 		return false;
 	}
 
-	// get measurement and correct for antenna array yaw offset
-	const float measured_yaw = _gps_sample_delayed.yaw + _gps_yaw_offset;
+	// GPS yaw measurement is alreday compensated for antenna offset in the driver
```

Comment:
Why do we then need the offset variable at all?

Reply:
We need it to compute the angle of the antenna array with the vertical -Down- axis: https://github.com/PX4/ecl/blob/97225fcb7431ddcb2ca1e2f72ea79043597ebea6/EKF/gps_yaw_fusion.cpp#L59-L66

Then, we're actually doing the fusion in the antenna frame, but there is no real advantage because the innovation is the same.

Reply:
So could we get rid of the offset in general?

Reply:
To compute the innovation, I would say yes. I also thing that we should be able to use the function `fuseYaw312` instead of having a new one here; we basically just fuse a yaw measurement.
However, before I do that, I would have the opinion of @priseborough .

Reply:
See https://github.com/PX4/ecl/pull/860